baseline.yaml

```
@@ -649,6 +649,26 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-17   
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project MUST perform a risk assessment to 
+       understand the most likely and impactful problems 
+       that could occur within the software.
```

```suggestion
       The project MUST perform a security risk assessment to 
       understand the most likely and impactful potential security problems 
       that could occur within the software.
```

We must clarify we're talking about security.

To some managers, a "risk assessment" for a project is determining the risks to failing to meet schedule and/or budget. Functionality and security might be gleefully traded away for schedule and budget. I don't think that's what you mean. Also say "potential... problem"; a risk is a potential problem, and if it's an actual problem it's no longer just a risk.